packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -293,6 +293,20 @@ export const DEFAULT_TRANSFORM_KEYS = [
   'rtcCandidatePairAudio',
   'rtcCandidatePairVideo'
 ];
+export const DEFAULT_FLATTEN_DATA_KEYS = [
```

Comment:
Would be great to add a comment above this to indicate where these constants come from and are expected to be used.